### PR TITLE
Updated event notifications in playground

### PR DESCRIPTION
**Pull request by 14gracel**

### CODE REVIEW

**Comment by dselman** [comment]
@caroline-church please review

File: packages/composer-playground/src/app/basic-modals/success/success.component.scss [hunk 1]
```diff
@@ -42,6 +42,13 @@ success {
                 height: 24px;
                 margin-right: $space-medium;
             }
+        
+            .transaction-link {
+                color:#3f55af!important;
```
Comment (caroline-church) [inline packages/composer-playground/src/app/basic-modals/success/success.component.scss]: Is there not a colour variable for this?, if not you should create one. Also do you really need the important?

File: packages/composer-playground/src/app/basic-modals/success/success.component.html [hunk 2]
```diff
@@ -13,6 +13,7 @@
         <div>
             <div class="notification-title">{{message.title}}</div>
             <p class="notification-text" [innerHTML]="message.text"></p>
+            <p *ngIf="events.length > 0"><a (click)="onEvent()" href="#" class="transaction-link">{{events.length}} event emitted</a></p>
```
Comment (caroline-church) [inline packages/composer-playground/src/app/basic-modals/success/success.component.html]: Can we make this more generalised incase we have different links in the future

File: packages/composer-playground/src/app/view-transaction/view-transaction.component.scss [hunk 3]
```diff
@@ -0,0 +1,121 @@
+@import '../../assets/styles/base/_colors.scss';
+@import '../../assets/styles/base/_variables.scss';
+@import '../../assets/styles/base/_typography.scss';
+
+.composer {
+  .transactionnav {
+    background-color: $third-highlight !important;
```
Comment (caroline-church) [inline packages/composer-playground/src/app/view-transaction/view-transaction.component.scss]: There are alot of !important in here, are they really all needed?

File: packages/composer-playground/src/app/test/view-transaction/view-transaction.component.scss [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                            border-right-color: $sixth-highlight;
+                            margin-top: 0;
+                            margin-bottom: 0;
+                        }
+                    }
+                    a {
+                        color: $primary-text;
+                    }
+                    .action {
+                        margin-left: 1rem;
```
Comment (caroline-church) [inline packages/composer-playground/src/app/test/view-transaction/view-transaction.component.scss]: this should be using variables

File: packages/composer-playground/src/app/test/view-transaction/view-transaction.component.scss [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+      }
+      .resource-content {
+        position: relative;
+        width: 100%;
+        background-color: $third-highlight;
+        padding: $space-small $space-medium;
+        text-align: left;
+        border-top:1px solid #e3ecec;
+        .CodeMirror {
+          height: 13vh;
```
Comment (caroline-church) [inline packages/composer-playground/src/app/test/view-transaction/view-transaction.component.scss]: don't use vh here other wise this causes problems when you change the size of your screen it should be in px

File: packages/composer-playground/src/app/test/view-transaction/view-transaction.component.scss [hunk 6]
```diff
@@ -24,8 +24,8 @@ transaction-modal {
                         color: $primary-text;
                     }
                     .action {
-                        margin-left: 1rem;
-                        margin-right: 1rem;
+                        margin-left: $space-smedium;
```
Comment (caroline-church) [inline packages/composer-playground/src/app/test/view-transaction/view-transaction.component.scss]: 1rem should be $space-medium

**Pull request merged**